### ISSUE TITLE
[TRACKING] Add Contribution guidelines to repository

### ISSUE THREAD

**Issue by jakobe** [kirbydesign/designsystem#1393]
**Please add a short description of your enhancement request**
I would like the Kirby repository to include contribution guidelines.

**Describe the solution you'd like**
Add a `.github/CONTRIBUTING.md` to the Kirby repo. This is the de facto standard for Github projects: https://docs.github.com/en/github/building-a-strong-community/setting-guidelines-for-repository-contributors

**Describe alternatives you've considered**
Leaving contributors in the dark and keep answering questions ad hoc ;)

**Additional context**
See Ionic's contribution guidelines for inspiration:
https://github.com/ionic-team/ionic-framework/blob/master/.github/CONTRIBUTING.md

- [ ] When these tasks are done, we also need to add a "How to contribute" section to the cookbook: #1373

# Tracking 
An outline has been created (see draft PR: https://github.com/kirbydesign/designsystem/pull/1404), and it has been decided that the separate sections of the contribution guidelines, should be split into separate issues such that they can be delegated. 

Also please speak up if you think there is anything that should be changed in the outline or you have anything to add 😄

## Sub-issues: 
- [X] [Documentation] Write "The contribution model"-section for contribution guidelines (#1421)
- [X] [Documentation] Write "Setup environment and tooling for contribution"-section for contribution guidelines (#1422)
- ~~[Documentation] Write "Creating pull-requests"-section for contribution guidelines (#1423)~~
- [X] [Documentation] Write "The good: issue"-section for contribution guidelines (#1424)
- [X] [Documentation] Write "The good: Branch"-section for contribution guidelines ( #1472)
- [X] [Documentation] Write "The good: commit"-section for contribution guidelines (#1425)
- [x] [Documentation] Write "The good: self-review"-section for contribution guidelines (#1426)
- [X] [Documentation] Write "The good: pull-request"-section for contribution guidelines (#1427)
- [ ] [Documentation] Write "The good: test"-section for contribution guidelines (#1428)
- [ ] [Documentation] Reading List for Kirby (#1556)
- [x] [Documentation] Write "Glossary"-section for contribution guidelines (#1429)

**note:** it might be a good idea to go through the feedback on the outline draft PR: #1404 

## Branching out 
Please branch out from the [`documentation/contribution-guidelines`](https://github.com/kirbydesign/designsystem/tree/documentation/contribution-guidelines) branch as this is where the outline exists in the file `.github/CONTRIBUTION.md`. 

## Creating PRs
When creating PRs for the contribution guidelines sub-issues: please use the [`documentation/contribution-guidelines`](https://github.com/kirbydesign/designsystem/tree/documentation/contribution-guidelines) branch as the base branch. When all sub-issues has been completed, [`documentation/contribution-guidelines`](https://github.com/kirbydesign/designsystem/tree/documentation/contribution-guidelines) will be merged to master.
![loop](https://user-images.githubusercontent.com/17269698/111777091-1a7f3580-88b3-11eb-99a6-614cfa88a492.gif)

## Discussion and feedback
The following draft pull-request can be used for general discussion and feedback of contribution guidelines: #1404 

## Tasks: 
[ ] Complete all sub-issues - all changes should now be on `documentation/contribution-guidelines`
[ ] https://github.com/kirbydesign/designsystem/pull/1404 should be moved from draft PR to a normal PR
[ ] https://github.com/kirbydesign/designsystem/pull/1404 should be reviewed; focusing on consistency, correctness and keeping the document DRY to the extents possible. Also we should discuss if some of the parts potentially should be moved to separate readme's etc. 
[X] Make a strategy for how to split up the contribution guidelines, focusing on how to get the information to contributors just in time. Avoiding having to read a huge blob of text before contributing. 
[ ] Have someone outside of team-kirby read the contribution guidelines and see if they feel like they are equipped to begin contributing. 
[ ] Merge PR and et voilà - we have contribution guidelines 🎉
[ ] We should align the pull-requests templates with the contribution guidelines. Making sure that there is no contradicting information, and have them be incorporated neatly.

**Comment by MadsBuchmann** [comment]
I have created the branch [`documentation/contribution-guidelines`](https://github.com/kirbydesign/designsystem/tree/documentation/contribution-guidelines) where an empty `.github/CONTRIBUTING.md`-file can be found. Feel free to add to this - however do not delete others text from this file. We should sift through it in the end, and see what sticks. Treat it as a brain-dump for now 🧠

Also I have assigned myself to this - but feel free to assign yourself as well, if you're working on it.
